nvm.sh

```diff
@@ -859,7 +859,6 @@ nvm_get_arch() {
   case "$NVM_UNAME" in
     *x86_64*) NVM_ARCH=x64 ;;
     *i*86*) NVM_ARCH=x86 ;;
-    *armv6l*) NVM_ARCH=arm-pi ;;
```

Comment:
You appear to be completely correct - perhaps for `armv61`, `armv71`, the `io.js` code can look for those names on the remote, and the `node` code can look for `arm-pi` on the remote?

That way, a meaningful platform-specific error message could be provided.